pyphare/pyphare/pharesee/run.py

```diff
@@ -57,8 +57,8 @@ def make_interpolator(data, coords, interp, domain, dl):
                                 assume_sorted=False)
 
         nx = 1+int(domain[0]/dl[0])
-        x = dl[0]*np.arange(0, nx)
 
+        x = yeeCoordsFor([0]*dim, [5]*dim, dl, [nx],  qty, "x")
```

Comment:
for coming reformating, we'd better have `[nbr_of_ghost]*dim` as second arg of `YeeCoordsFor`

Reply:
changer